qualtran/bloqs/mod_arithmetic/mod_addition.py

```diff
@@ -269,6 +271,15 @@ class CModAddK(Bloq):
     Registers:
         ctrl: The control bit
         x: The register to perform the in-place modular addition.
+
+    References:
```

Comment:
refactor from (refs)x(commentary) to zip(refs, commentary)

```
[arithmetic](...)
Author. yyy.

[rsa](...).
Gidney and Ekera 2019. 
Uses CModAddK in the "Reference implementation" using ref 1, fig 4 etc etc

[ecc](...).
Litinski et al. yyyy.
Uses 2 fewer add operations because K is classical (??)
```